misk/src/main/kotlin/misk/clustering/ClusterResourceMapper.kt

```diff
@@ -9,7 +9,7 @@ package misk.clustering
 interface ClusterResourceMapper {
   /**
    * @throws NoMembersAvailableException if there are no members in the cluster
-   * @return The [Cluster.Member] that should own the given resource id
+   * @return A map of resource IDs to [Cluster.Member].
    */
-  operator fun get(resourceId: String): Cluster.Member
+  fun getMappings(resourceIds: Set<String>, members: Set<Cluster.Member>): Map<String, Cluster.Member>
```

Comment:
> Because Range ResourceMapper makes assignments in batches, this PR also makes ClusterResourceMapper return a set of mappings instead of a single entry.

I don't understand, could you explain this?